packages/dev/src/CustomCatalog.tsx

```diff
@@ -80,9 +81,46 @@ export const CustomCatalog: React.FC = () => {
     setFilteredQuickStarts(result);
   };
 
+  const [bookmarked, setBookmarked] = React.useState<string[]>([])
+
   const CatalogWithSections = React.useMemo(
     () => (
       <>
+        <QuickStartCatalogSection>
+          <TextContent>
+            <Text component="h2">Bookmarkable</Text>
+            <Text component="p" className="catalog-sub">
+              Bookmarkable examples
+            </Text>
+          </TextContent>
+          <Gallery className="pfext-quick-start-catalog__gallery" hasGutter>
+            {allQuickStarts
+              .filter((quickStart: QuickStart) => quickStart.metadata.instructional)
+              .map((quickStart: QuickStart) => {
+                const {
+                  metadata: { name: id },
+                } = quickStart;
+
+                return (
+                  <GalleryItem key={id} className="pfext-quick-start-catalog__gallery-item">
+                    <QuickStartTile
+                      action={{
+                        onClick: (e: React.SyntheticEvent) => {
+                          e.stopPropagation();
```

Comment:
The entire tile is clickable so I had to prevent the propagation. I was thinking of adding this directly to the button in header, but that felt too strict. Let me know if you'd prefer it to be there or not.

Reply:
what you have works for me.